apps/service_providers/llm_service/state.py

```diff
@@ -218,3 +219,12 @@ def save_message_to_history(self, message: str, type_: ChatMessageType, annotati
             content=message,
             metadata={"openai_file_ids": annotation_file_ids} if annotation_file_ids else {},
         )
+
+
+class AssistantAgentState(AssistantExperimentState):
```

Comment:
Is this class even necessary? Seems like you could just put these on the `AssistantExperimentState` class.